### PR TITLE
LRCI-2522 Update ocular method for use in Poshi

### PR DESCRIPTION
**Pull request by CalumR23**
https://issues.liferay.com/browse/LRCI-2522

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

File: modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java [hunk 1]
```diff
@@ -4540,13 +4542,15 @@ protected boolean isValidKeycode(String keycode) {
 	}
 
 	protected void ocularConfig() {
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java:4544]: Call this in the constructor of BaseWebDriverImpl

File: modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java [hunk 2]
```diff
@@ -2197,14 +2197,16 @@ public Navigation navigate() {
 	}
 
 	@Override
-	public void ocularAssertElementImage(String locator) throws Exception {
+	public void ocularAssertElementImage(String locator, String filePath)
+		throws Exception {
+
 		ocularConfig();
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java:2203]: Remove this, and replace with: 
```
OcularConfiguration ocularConfiguration = Ocular.config();

ocularConfiguration.resultPath(Paths.get(testBaseDirName, getOcularResultImageDirName())); // make this specific to the image
```

At the end: 
```
ocular
    snap
         path
               image1.png
    result (not committed) 
         path
               image1.png
```

File: modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java [hunk 3]
```diff
@@ -4540,13 +4542,15 @@ protected boolean isValidKeycode(String keycode) {
 	}
 
 	protected void ocularConfig() {
+		String testBaseDirName = PropsValues.TEST_BASE_DIR_NAME;
+
 		OcularConfiguration ocularConfiguration = Ocular.config();
 
 		ocularConfiguration = ocularConfiguration.snapshotPath(
-			Paths.get(".", getOcularSnapImageDirName()));
+			Paths.get(testBaseDirName, getOcularSnapImageDirName()));
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java:4551]: Create the result dir. 
```
File file = new File(Paths.get(testBaseDirName, getOcularResultImageDirName()));

file.mkdirs();
```

**Review by kenjiheigel — COMMENTED** [review]
Create Poshi ticket, and recommit under that ticket.

Remove committed png files copied from Sikuli.

**Pull request closed without merging**